scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -54,7 +54,7 @@ address = "0.0.0.0:9191"
 username = "root"
 password = "root"
 client_timeout_in_second = 60
-auto_sync_interval = 10
+auto_sync_interval = 60
```

Comment:
It seems in the Config `auto_sync_interval` should be a type of `Option<u64>`

Reply:
I think this is handled [here](https://github.com/datafuselabs/databend/pull/6187/files#diff-4a92ce9bc7aa54fb9220136632a7ea6e16bf9648eeef440f47eb35e350a39864R318)